File: javascript/tools/latn_iast_transliteration_verification/deva-to-gujr-test.js
import { devanagariSmokeSamples } from '../../src/example-deva.js';
import { toCanonicalGujaratiFromDevanagari } from '../../src/deva-gujr-converter.js';

console.log('----------------------------------------------------------------');
console.log(' DEVANAGARI TO GUJARATI SCRIPT CONVERSION');
console.log('----------------------------------------------------------------');

for (const source of devanagariSmokeSamples) {
    const result = toCanonicalGujaratiFromDevanagari(source);
    console.log(`"${source}" -> "${result}"`);
}

File: dart/tool/latn_iast_transliteration_verification/deva_to_gujr_test.dart
// ignore_for_file: avoid_print

import 'dart:convert';

import 'package:lipimala/deva_gujr_converter.dart';
import 'example_deva.dart' as examples;

void main() {
  print('----------------------------------------------------------------');
  print(' DEVANAGARI TO GUJARATI SCRIPT CONVERSION');
  print('----------------------------------------------------------------');

  for (final source in examples.devanagariSmokeSamples) {
    final result = source.toCanonicalGujaratiFromDevanagari();
    print('"$source" -> "$result"');
  }
}
